### ISSUE TITLE
Syntax support request: Path

### ISSUE THREAD

**Issue by thekip** [anboralabs/intellij-firebase-highlighter#39]
Snippet:

```
rules_version = '2';
service cloud.firestore {
  match /databases/{database}/documents {
    match /{path=**}/{item} {
      allow read: if path != /users;
    }
  }
}
```

Firestore Rules supports [Path datatype ](https://firebase.google.com/docs/reference/rules/rules.Path) which is created as

> "raw" form beginning with a forward slash /

``` 
/path/to/resource
```

## Actual result

The plugin doesn't support this syntax and breaks highlighting on them, and on the rest of the document

![image](https://user-images.githubusercontent.com/1586852/105464853-7900ac80-5c92-11eb-9893-4b7a299fc5a4.png)

## Expected result

Plugin should support this syntax

Please note, the '/users' and /users is not the same, fisrts one is a String where the second one is Path.  Comparison from the snippet above would always be falsy if compared to string ex: ` if path != '/users';`

**Comment by dalgarins** [comment]
<img width="932" alt="Captura de Pantalla 2021-01-23 a la(s) 9 47 40 a  m" src="https://user-images.githubusercontent.com/8527174/105595468-702adc00-5d61-11eb-8ddc-657900071bbd.png">

Thanks for testing this issue will be fixed on version: 2.2.0

**Comment by thekip** [comment]
Thanks!
